cmd/cerbosctl/hub/store/add_files.go

```diff
@@ -0,0 +1,204 @@
+// Copyright 2021-2025 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package store
+
+import (
+	"context"
+	"fmt"
+	"io/fs"
+	"iter"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/alecthomas/kong"
+
+	"github.com/cerbos/cerbos-sdk-go/cerbos/hub"
+	"github.com/cerbos/cerbos/internal/util"
+	storev1 "github.com/cerbos/cloud-api/genpb/cerbos/cloud/store/v1"
+)
+
+const addFilesHelp = `
+The following exit codes have a special meaning.
```

Comment:
Is a non-zero exit code going to complicate running this in CI?

I don't know what is best practice here, but I wouldn't expect my release pipeline to fail if the policies haven't changed.

Reply:
Seconded. 5 here would flag  by lots of things (normal CI usage, shells). 6 is OK as things should fail in that case. If you want behaviour like 5, I'd suggest adding an extra flag "--error-on-unchanged", possibly taking an optional code to use as the exit (could default to 5).

Reply:
Another alternative would be to have an standardised yaml output that could include other fields to indicate state beyond exit. However that would probably be something you;d want to be consistant across all commands

Reply:
The no-op case is not straightforward to handle because the API doesn't return the store version when it's a no-op. The output from the command wouldn't be consistent in that case anyway. I'll figure out a fix for that in a different PR. Until then, you can do `cerbosctl ... || true` as a workaround.